lib/srv/discovery/access_graph.go

```diff
@@ -211,6 +216,11 @@ func (s *Server) initializeAndWatchAccessGraph(ctx context.Context, reloadCh <-c
 	 }`
 	)
 
+	clusterFeatures := s.Config.ClusterFeatures()
+	if !clusterFeatures.AccessGraph && (clusterFeatures.Policy == nil || !clusterFeatures.Policy.Enabled) {
```

Comment:
Is it possible to have AccessGraph but not Policy.Enabled? 🤔

Reply:
legacy features. Will be removed soon